### ISSUE TITLE
Cypress server fails to start

### ISSUE THREAD

**Issue by autumnwoodberry** [LinusBorg/vue-cli-plugin-test-attrs#7]
Following the basic instructions to install in the readme but unable to get cypress working. Verified same error occurs on newly created vue cli application with default settings + cypress. Terminal is not showing any errors, but cypress service window itself doesn't load, and shows an error:

```
 Can't start server
The function exported by the plugins file threw an error.

We invoked the function exported by '/Users/adamwoodbury/playground/strawberry/tests/e2e/plugins/index.js', but it threw an error.

The following error was thrown:

/Users/adamwoodbury/playground/strawberry/node_modules/vue-cli-plugin-test-attrs/index.js:23
const options = { ...defaultOptions, ...pluginOptions }
^^^

SyntaxError: Unexpected token ...
at createScript (vm.js:74:10)
at Object.runInThisContext (vm.js:116:10)
at Module._compile (module.js:533:28)
at Object.Module._extensions..js (module.js:580:10)
at Module.load (module.js:503:32)
at tryModuleLoad (module.js:466:12)
at Function.Module._load (module.js:458:3)
at Module.require (module.js:513:17)
at require (internal/module.js:11:18)
at idToPlugin (/Users/adamwoodbury/playground/strawberry/node_modules/@vue/cli-service/lib/Service.js:136:14)
at Object.keys.concat.filter.map.id (/Users/adamwoodbury/playground/strawberry/node_modules/@vue/cli-service/lib/Service.js:176:20)
at Array.map (native)
at Service.resolvePlugins (/Users/adamwoodbury/playground/strawberry/node_modules/@vue/cli-service/lib/Service.js:162:10)
at new Service (/Users/adamwoodbury/playground/strawberry/node_modules/@vue/cli-service/lib/Service.js:33:25)
at Object. (/Users/adamwoodbury/playground/strawberry/node_modules/@vue/cli-service/webpack.config.js:8:13)
at Object. (/Users/adamwoodbury/playground/strawberry/node_modules/@vue/cli-service/webpack.config.js:14:3)
at Module._compile (module.js:569:30)
at Object.Module._extensions..js (module.js:580:10)
at Module.load (module.js:503:32)
at tryModuleLoad (module.js:466:12)
at Function.Module._load (module.js:458:3)
at Module.require (module.js:513:17)
```

**Comment by LinusBorg** [comment]
Hey @autumnwoodberry 

thanks for reporting. On which version did that happen? Given the date you openend this issue, it was before 3.1.2 was released.

We removed the cypress webpack-preprocessor in that release because it was causing too much trouble, and this is probably one of those moments.

Cypress uses a different node version, and I think it'a old enough to not support object rest spread. So when its webpacl-preprocessor plugin tries to load the webpack config, it fails because of that? Even though the error message looks like it was complaining about `const` ...

So we can do two things:

1. Test again with the latest release where no webpack-preprocessor is included.
2. If that still fails, I can try and fix the code.
